ext/scheduler/airflow2/resources/base_dag.py

```diff
@@ -43,7 +43,8 @@
     schedule_interval={{ if eq .Job.Schedule.Interval "" }}None{{- else -}} {{ .Job.Schedule.Interval | quote}}{{end}},
     sla_miss_callback=optimus_sla_miss_notify,
     catchup={{ if .Job.Behavior.CatchUp -}}True{{- else -}}False{{- end }},
-    dagrun_timeout=timedelta(seconds=DAGRUN_TIMEOUT_IN_SECS)
+    dagrun_timeout=timedelta(seconds=DAGRUN_TIMEOUT_IN_SECS),
+    tags=[{{.Job.Labels | values | join '", "' | quote }}]
```

Comment:
job labels is in a `map[string]string` type, not sure it can be joined correctly this way (tested and it failed). how about having similar like this?
```python
tags = [
    {{- range $key, $value := $.Job.Labels}}
    "{{ $key }}:{{ $value }}",
    {{- end}}
]
```
or let me know if you find a better way to do it.

Reply:
files updated, need team to test it again.
https://github.com/odpf/optimus/pull/156/commits/e7100af1f1212596c6b25adde4c1631c45b493d7